mne_bids/mne_bids.py

```diff
@@ -652,7 +746,7 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     # Re-save FIF files to fix the file pointer for files with multiple parts
     # This is WIP, see: https://github.com/mne-tools/mne-python/pull/5470
     if ext in ['.fif']:
-        raw.save(raw_file_bids, overwrite=overwrite)
+        raw.save(raw_file_bids, overwrite=(write_mode == 'overwrite'))
```

Comment:
Here, if `write_mode == 'error'`, mne won't throw an error. Instead it will block execution and ask the user if we should overwrite or not. I think we should throw the error manually since mne does not do it.

Reply:
Ah ok. This is not clear from the docstring for the `.save` method
> If False (default), an error will be raised if the file exists. 

I'll raise an error manually instead